### ISSUE TITLE
Some Command Line Arguments Not Working

### ISSUE THREAD

**Issue by VShrubowich** [Microsoft/vscode#36182]
- VSCode Version: Code 1.17.1 (1e9d36539b0ae51ac09b9d4673ebea4e447e5353, 2017-10-10T14:24:50.851Z)
- OS Version: Windows_NT x64 10.0.15063
- Extensions: 

Extension|Author (truncated)|Version
---|---|---
PowerShell|ms-|1.4.3

---

Steps to Reproduce:

1.  The following command line just opens the VS Code GUI window:  code -v

Reproduces without extensions: Yes

---

The command line arguments that are supposed to output to the console are not working.  Examples include -h or --help, -v or --version, --list-extensions, and --show-versions.

This is what I get when I execute 'code --list-extensions --show-versions --verbose -v -h':

[main 9:12:53 PM] Starting VS Code in verbose mode
[main 9:12:53 PM] from: c:\Program Files\Microsoft VS Code\resources\app
[main 9:12:53 PM] args: { _: [],
  help: true,
  h: true,
  version: true,
  v: true,
  wait: false,
  w: false,
  diff: false,
  d: false,
  add: false,
  a: false,
  goto: false,
  g: false,
  'new-window': false,
  n: false,
  'unity-launch': false,
  'reuse-window': false,
  r: false,
  performance: false,
  p: false,
  'prof-startup': false,
  verbose: true,
  logExtensionHostCommunication: false,
  'disable-extensions': false,
  disableExtensions: false,
  'list-extensions': true,
  'show-versions': true,
  nolazy: false,
  'skip-getting-started': false,
  'sticky-quickopen': false }
[main 9:12:54 PM] IPC#vscode-machineId
[main 9:12:57 PM] IPC#vscode-workbenchLoaded

At this point, VS Code opens.

**Comment by joaomoreno** [comment]
Make sure you don't have `Code.exe` in your `PATH`, but `code.bat`, which is in the `bin` directory of VS Code. What is your full `PATH` environment variable value?

**Comment by vshrubowich** [comment]
That was it.  Thanks.  I run as a non-admin user but had to install VS Code as an admin user.  The admin user's PATH was updated but not the non-admin user's PATH.  This may be because I had the 32-bit version installed before, which was in the non-admin user's PATH, then switched to the 64-bit version.  If I get a chance, I'll grab a spare computer and repeat the 32-bit to 64-bit switch and see what happens.

I didn't realize that I should be running code.cmd instead of code.exe so I was switching to "C:\Program Files\Microsoft VS Code" and running "code" from there.

I updated PATH in the system environment variables and all is good now.  I don't know if this is still a bug (PATH not updating for the non-admin user) or if it should now be a feature request to have the VS Code installer update the system PATH variable instead of the user PATH variable.
